docs/content_management/pages/page_builder_guide.md

```diff
@@ -0,0 +1,213 @@
+
+![Switch layout](switch_layout_window.png)
+
+Applying a layout divides the Page into the defined zones. The zones are placeholders for Content items.
+
+On the Page creation modal, select the layout and click **Create draft**.
+Now you are ready to add blocks of content to the Page.
+
+The page layouts that an editor has access to are up to you to choose.
+In the `Select layouts` section, you can select layouts that you want to be available for the Page.
+
+The default, built-in Page layout has only one zone, but developers can create other layouts in configuration.
+For more information, see [Configure layout](https://doc.ibexa.co/en/latest/templating/render_content/render_page/#configure-layout).
+
+#### Add blocks
+
+To customize your page in Page Builder you need to add blocks.
+To do it, access Elements toolbar, drag page block that you want to use and drop it on the empty place on a drop zone.
+
+When you add a new block to the drop zone, before you drop it, a blue line appears  - it helps you see the position of the newly added block in relation to other, already added blocks.
+
+![Drop zone line](drop_zone_line.png)
+
+Ready-to-use blocks available in [[= product_name =]] have their own, unique functions, but you can also [add your own, custom blocks](create_custom_page_block.md). All available tools and settings, that Page Builder comes with, enable you to customize the content appearing on the page.
+
+You can check all ready-to-use blocks available in Page Builder in User Documentation, [Block reference page](https://doc.ibexa.co/projects/userguide/en/latest/content_management/block_reference/).
+
+#### Work with blocks
+
+Working with blocks is intuitive. You don't have to worry about placing blocks in the proper place from the start - you can reorder the at any time.
+You can reorder blocks in a few ways:
+
+- drag and drop block in the desired location on a drop zone
+- select block and use up and down arrow on the keyboard
+- access Structure view and use 'Move up' and 'Move down' function in the settings of the block or drag and drop to change the position in the structure
+
+![Structure view - drag and drop](structure_view_drag_drop.png)
+
+You can manage each block by accessing its settings. To do it, click settings icon next to the block's name.
+
+![Block settings](block_settings.png)
+
+Available settings are:
+
+- Move up - allows you to change position of the block on the page by moving it up
+- Move down - allows you to change position of the block on the page by moving it down
+- Configuration - allows you to access configuration window
+- Duplicate - duplicates existing block by creating its copy
+- Refresh - refreshes preview of the block
+- Delete - deletes existing block
+
+#### Schedule content
+
+Page Builder comes with a Scheduler - is a tool that allows you to schedule content publication.
```

Comment:
This one verb "is" sounds weird here as not in a definition list.
I would also avoid "publication" that could mislead to a confusion with "publish" concept.
```suggestion
Page Builder comes with a Scheduler - a tool that allows you to schedule content appearance.
```